feature-utils/poly-look/storybook/.storybook/react/preview.jsx

```diff
@@ -21,7 +21,9 @@ const withTheme = (Story, context) => {
     }
     default: {
       return (
-        <div className={`poly-theme poly-theme-${theme}`}>
+        <div
+          className={`poly-theme poly-theme-${theme} theme-holder single-holder`}
+        >
```

Comment:
These divs are wrappers for all the stories. And because we now have a variable for the background color, I wanted to use that as well, to make them better looking and to see how it they would look like against a certain background. But when picking a single theme, the container wasn't full screen so it looked bad. I've tried to make it full screen so it looks better xD